open-company/open-company-web
# Settings restyle

Card: https://trello.com/c/c1I6X6Vp

Abstract: https://share.goabstract.com/51827e96-b0c6-4284-b224-5850bff59aca

Related PR:
- Auth: https://github.com/open-company/open-company-auth/pull/42

To test:
- open digest settings from the menu
- hover the company logo
- [x] do you see the tooltip? Good
- click to upload a new logo
- cancel out of the FileStack picker
- click Save at the bottom
- [x] do you have an org without a logo now? Good
- go back into digest settings
- upload a new logo
- change the digest name
- Save and refresh
- add an email domain and refresh
- remove the email domain and refresh
- add a Slack org (no bot)
- add the bot to the slack org
- remove the slack org
- switch to manage memebers
- change the role of some users
- switch to invite users
- invite some email users
- invite some slack users
- go to manage members tab
- resend some invitatinos
- remove at least one invited user


### NB: ping Sean once this is out to beta to run the script to retrieve Slack team avatar and domain for the slack-org table of auth DB
